src/screens/LeaderboardScreen.js

```diff
@@ -27,46 +53,45 @@ export class LeaderboardScreen extends Component<Props> {
   };
 
   render() {
+    const { isFetching } = this.state;
+
     return (
       <Subscribe to={[RootStore]}>
         {rootStore => (
           <FlatList
-            style={{ marginTop: 20 }}
-            refreshing={this.state.isFetching}
+            style={styles.container}
+            refreshing={isFetching}
             onRefresh={() => this._onRefresh(rootStore)}
             data={rootStore.state.leaderboard}
             keyExtractor={item => JSON.stringify(item.id)}
-            renderItem={({ item, index }) => (
-              <View
-                style={{
-                  flex: 1,
-                  alignSelf: "stretch",
-                  flexDirection: "row",
-                  padding: 15,
-                  borderBottomWidth: 3,
-                  borderBottomStartRadius: 15,
-                  borderBottomEndRadius: 15
-                }}
-              >
-                <View style={{ flex: 2 }}>
-                  <TouchableOpacity
-                    onPress={() =>
-                      alert(`W: ${item.wins} L: ${item.games - item.wins}`)
-                    }
-                  >
-                    <Text>
-                      {index + 1}. {item.commander}
-                    </Text>
-                  </TouchableOpacity>
-                </View>
-                <View style={{ flex: 1, flexDirection: "row-reverse" }}>
-                  <Text>{Number(item.elo).toFixed(2)}</Text>
-                </View>
-              </View>
-            )}
+            renderItem={({ item, index }) =>
+              this._renderLeaderBoard(item, index)
+            }
           />
         )}
       </Subscribe>
     );
   }
 }
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 20
+  },
+  leaderboard: {
+    flex: 1,
+    alignSelf: "stretch",
+    flexDirection: "row",
+    padding: 15,
+    borderBottomWidth: 3,
+    borderBottomStartRadius: 15,
+    borderBottomEndRadius: 15
+  },
+  leaderboard__deckName: {
+    flex: 2
+  },
+  leaderboard__deckElo: {
+    flex: 1,
+    flexDirection: "row-reverse"
+  }
+});
```

Comment:
Nice job on adding a `StyleSheet`!

Some people like to have a folder for each `component` in the `/components` folder, and then have a file named `index.js` and a file called `style.js`